classes/class-p4-post.php

```diff
@@ -156,28 +164,76 @@ public function get_author_override() {
 		}
 
 		/**
-		 * Get post/page custom planet4 type.
+		 * Gets the tags for this P4_Post.
+		 */
+		public function set_tags() {
+			$tags = wp_get_post_tags( $this->id );
```

Comment:
@angtheod you can get the post tags with timber also.
https://timber.github.io/docs/reference/timber-post/#terms
It returns Timber\Term objects which contain the tag link also.
https://timber.github.io/docs/reference/timber-term/#link

Reply:
I agree overall, but then I will need to change also the way that the controller passes those data during page load to the template. Should I do this in this PR?

Reply:
No. It is outside the scope of this. Please open a new jira issue for it.

Reply:
Yes, you are right. it needs change there also because we use arrays there.
Because is big change, let's address it in a different issue.